pkg/apis/forklift/v1beta1/plan/vm.go

```diff
@@ -30,6 +30,8 @@ type VM struct {
 	ref.Ref `json:",inline"`
 	// Enable hooks.
 	Hooks []HookRef `json:"hooks,omitempty"`
+	// Choose the bootable disk number for praimery
+	RootDisk string `json:"rootDisk,omitempty"`
```

Comment:
that's an interesting decision to place this field at the vm level
on the one hand, we do the same for hooks but as it goes against out desire to have the UI designed for scale migrations, the UI allows users to specify a hook per plan and then sets it per each vm in the plan
on the other hand, [we considered this for encryption keys](https://github.com/kubev2v/forklift/pull/676) and decided to place it at the plan-level

we better be consistent here
I don't mind going with this approach of specifying the primary disk per-VM but then we should also do that for the encryption keys and implement this field at the UI level as hooks
@liranr23 WDYT?

Reply:
@ahadas I do think its better to keep it in the VM level (for all cases) its giving more flexibility when setting up planes for case we want to have more that one VM like this in a plan, otherwise the user will need to have separate plans for each VM.

Reply:
we do have something different.
here, we need to provide per VM the boot device.
in LUKS, we also need the pass phrase per device, but since the device id is unique, we can put all the ids and their pass phrases to every virt-v2v command. it will just ignore the non-existing devices. therefore, we can ease on the process, and generalize it to the plan (make it more mass migration state of mind).

i do think the boot disk makes more sense to be per-vm, but i don't think we need to change the LUKs approach, especially we will have the `all` devices option from virt-v2v.

Reply:
@liranr23 ok, so I commented on the encryption keys part in #676 - we can continue that there
so if we're in agreement that it's better to place the primary disk configurations like hooks, let's go with that then